tests/conftest.py

```diff
@@ -113,6 +113,28 @@ def _x_series_device(device_type, system):
     return None
 
 
+def _field_daq_device(device_type, system):
+    for device in system.devices:
+        device_type_match = (
+            device_type == DeviceType.ANY
+            or (device_type == DeviceType.REAL and not device.is_simulated)
+            or (device_type == DeviceType.SIMULATED and device.is_simulated)
+        )
+        if (
+            device_type_match
+            and device.product_category == ProductCategory.FIELD_DAQ
+            and len(device.ai_physical_chans) >= 1
+        ):
+            return device
+
+    pytest.skip(
+        "Could not detect a device that meets the requirements to be an Field DAQ fixture of type "
+        f"{device_type}. Cannot proceed to run tests. Import the NI MAX configuration file located "
+        "at nidaqmx\\tests\\max_config\\nidaqmxMaxConfig.ini to create these devices."
+    )
+    return None
```

Comment:
I think we were a little too loose with these methods. For time-aware, specifically, a real device might act quite a bit differently than a simulated one. I would recommend you use the new method I added `_device_by_product_type` and implement a fixture called `sim_fdaq_device` or maybe even `sim_time_device`

Reply:
Agreed, but please spell out `field_daq` instead of abbreviating it `fdaq`.